addons/website/views/snippets/s_numbers_list.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_list" name="Numbers list">
+    <section class="s_numbers_list o_cc o_cc1 pt80 pb80">
+        <div class="container">
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-5 pb32">
+                    <h3>Key Metrics of Company's Achievements</h3>
+                    <p><br/>From revenue growth to customer retention and market expansion, our key metrics of company achievements underscore our strategic prowess and dedication to driving sustainable business success.</p>
+                </div>
+                <div class="col-lg-6 offset-lg-1 s_col_no_bgcolor" data-name="Numbers group">
+                    <div class="row">
+                        <div class="col-6 col-lg-6 pt16 pb16" data-name="Number">
+                            <h6 class="display-6">15%</h6>
```

Comment:
Sorry, I do not see how to obtain `<h6 class="display-6">` using the editor.
The closest I get is:
```xml
<h6 class="o_default_snippet_text"><span class="h2-fs">1 billion</span></h6>
```
which has the same font-size, but is unfortunately thicker.

Reply:
( @chgo-odoo is off for a few days. Helping him here. )
Closest I got was just with a `<span class="h2-fs">15%</span>` (no headings), less thicker. 

Would you agree with this solution?